dqc/system/mol.py

```diff
@@ -294,6 +294,27 @@ def getparamnames(self, methodname: str, prefix: str = "") -> List[str]:
         else:
             raise KeyError("Unknown methodname: %s" % methodname)
 
+    def make_copy(self, **kwargs) -> "Mol":
```

Comment:
Could you please add at the very top `from future import __annotations__` so the return type can be just `Mol` (without a double quote). It is just a stylistic preference and could also help users to navigate the code from github.

Reply:
good call out - i prefer this too - done using `from __future__ import annotations`